apps/condo/domains/subscription/components/SubscriptionPane.tsx

```diff
@@ -0,0 +1,159 @@
+    ]
+}
+
+export const SubscriptionPane: React.FC = () => {
+    const { isSmall } = useResponsive()
+    const intl = useIntl()
+
+    const { isVisible, setIsVisible, SubscriptionPaymentModal } = useSubscriptionPaymentModal()
+    const subscriptions = getSubscriptionOptions(intl, setIsVisible)
+    const activeSubscription = getActiveSubscriptionData(intl)
+
+    return (
+        <>
+            <Row gutter={[0, 60]}>
+                <Col span={24}>
+                    <Row gutter={[0, 20]}>
+                        <Col span={24}>
+                            <Typography.Title level={3}>
+                                {intl.formatMessage({ id: 'subscription.data.tariffs' })}
+                            </Typography.Title>
+                        </Col>
+                        <Col span={24}>
+                            <Row gutter={[0, 20]}>
+                                {subscriptions.map((subscription, index) => {
+                                    return (
+                                        <Col sm={24} lg={8} offset={isSmall || (index % 2 === 0) ? 0 : 1 } key={index}>
+                                            <FocusContainer margin={'0'}>
+                                                <Row gutter={[0, 20]}>
+                                                    <Col span={24}>
+                                                        <Typography.Paragraph>
+                                                            <Row gutter={[0, 8]}>
+                                                                <Col span={24}>
+                                                                    {subscription.renderTitle()}
+                                                                </Col>
+                                                                <Col>
+                                                                    <Typography.Text>
+                                                                        {subscription.renderDescription()}
+                                                                    </Typography.Text>
+                                                                </Col>
+                                                            </Row>
+                                                        </Typography.Paragraph>
+                                                    </Col>
+                                                    <Col span={24}>
+                                                        {subscription.renderAction()}
+                                                    </Col>
+                                                </Row>
+                                            </FocusContainer>
+                                        </Col>
+                                    )
+                                })}
+                            </Row>
+                        </Col>
+                    </Row>
+                </Col>
+                <Col span={24}>
```

Comment:
I think, we should render this conditionally, if `activeSubscription` is present.

Reply:
I will do it after merge of `ServiceSubscription` schema.